services/gcp/datastore/databases.yml

```diff
@@ -0,0 +1,46 @@
+name: Databases
+description: >-
+  Databases are the top level containers for data in Datastore mode. A datastore mode database is a type of database 
+  that uses the Google Cloud Datastore API to access data stored in Firestore databases.
+scope: MEDIUM
+notes: >-
+  Datastore mode quotas can be viewed in the GCP console at https://console.cloud.google.com/appengine/quotadetails.
+privileges:
+  get:
+    vulnerabilities: []
+    notes: >-
+      Principal with this permission can begin and rollback transaction. the privilege by itself has no impact, given transactions are isolated.
+  getMetadata:
+    vulnerabilities: [discovery:metadata]
+    notes: >-
+      Principal with this permission can get metadata of the database.
```

Comment:
Grammar:
```suggestion
      A principal with this permission can read the metadata of the database.
```

Also, is there anything sensitive in these metadat?

Reply:
i have verified, it does not share anything sensitive apart from mode of operation , data archival mode.